src/sql/workbench/browser/modal/modal.ts

```diff
@@ -71,6 +77,7 @@ export interface IModalOptions {
 
 const defaultOptions: IModalOptions = {
 	isFlyout: true,
```

Comment:
no need to keep the isFlyout property, it has never been exposed to external developers, no need to make it backward compatible.

Reply:
Removed. Also made sure to modify instances were isFlyout is supplied a value to now do so for dialogStyle.